README.md

```diff
@@ -20,6 +20,16 @@ fi
 Optionally, installation in a custom location is supported by setting and
 exporting `DVM_ROOT` in your environment prior to the above invocation.
 
+## Quick start
+
+```sh
+# these are all valid
+dvm install 3.4.1
+dvm install 3.5.0-164.0.dev
+dvm install latest
+dvm install latest dev
+```
+
```

Comment:
I'd prefer not to add a section specific to just install but agree it's probably useful to provide a bit more detail somewhere. Let's drop this section and instead, I'll add a `dvm help` command that dumps more detailed usage for each command and find a way to incorporate that into the README.